we have to set the cargo features manually, so allow an env override
diff --git a/setup.py b/setup.py
index 1c15258..4e22cfd 100644
--- a/setup.py
+++ b/setup.py
@@ -10,6 +10,7 @@
 # for maturin itself use `maturin sdist`.
 
 import platform
+import os
 import sys
 
 try:
@@ -74,7 +75,7 @@ setup(
     python_requires=">=3.7",
     cmdclass={"bdist_wheel": bdist_wheel},
     packages=["maturin"],
-    rust_extensions=[RustBin("maturin", args=cargo_args)],
+    rust_extensions=[RustBin("maturin", args=["--no-default-features", os.getenv("MATURIN_FEATURES")])],
     classifiers=[
         "Topic :: Software Development :: Build Tools",
         "Programming Language :: Rust",
